install_config/install/advanced_install.adoc

```diff
@@ -798,17 +798,57 @@ openshift_examples_modify_imagestreams=true
 |Set to `true` if pointing to a registry other than the default. Modifies the image stream location to the value of `*oreg_url*`.
 
 |`*openshift_docker_additional_registries*`
-|Specify the additional registry or registries.
-If the registry required to access the registry is other than `80` include the port number required.
+|Specify the additional registry or registries. If the registry required to access the registry is other than `80` include the port number required.
```

Comment:
@sosiouxme This doesn't look right. It is a change I made in a previous PR [1], but I didn't see it then. Is this is not right, @bfallonf , can you fix? 
_If the registry required to access the registry"
Should this be _If the **port** required to access the registry..._
[1] https://github.com/openshift/openshift-docs/pull/8314/files

Reply:
@mburke5678 This could make sense. The only thing I am questioning is that it's about "Additional registries", but that doesn't mean it's correct. @sosiouxme thoughts?

Reply:
Yes, s/registry/port/ there.

However it may be moot for 3.9+ at this point as all the docker config has moved to the prerequisites.yaml playbook so I'm not sure this goes here any more (might be a nice patch for older versions though).

Also now I doubt the default port is 80. I think it's either 443 (HTTPS for secure communication) or 5000 (docker default). I can't see how it would hurt to be explicit, so I'd probably change the whole line to "Specify the additional registry or registries in the form of <address>:<port>"

Reply:
@sosiouxme Sure. Changed to "Specify the additional registry or registries in the form of
<address>:<port>"